packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -1078,14 +1080,6 @@ export function sendTransaction<
 							ETH_DATA_FORMAT,
 						);
 
-						if (web3Context.handleRevert) {
-							// eslint-disable-next-line no-use-before-define
-							await getRevertReason(
-								web3Context,
-								transactionFormatted as TransactionCall,
-							);
-						}
-
```

Comment:
This is part of addressing #5839, the revert reason is retrieved after the transaction has been processed by the network and when `transactionReceipt.status === BigInt(0)`